xen-bugtool

```diff
@@ -426,6 +427,7 @@ cap(CAP_PERSISTENT_STATS,    PII_NO,                    max_size=50*MB,
     max_time=60, checked=False, hidden=True)
 cap(CAP_PROCESS_LIST,        PII_YES,                   max_size=30*KB,
     max_time=60)
+cap(CAP_SYSTEM_LOAD,         PII_MAYBE,                 max_size=70*MB, max_time=30)
```

Comment:
Should we remove this max_size from the CAP_XENSERVER_LOGS to avoid growing the size of the status report?

Reply:
For CAP_SYSTEM_LOAD: The files in /var/log/sa which are collected in it now is ~33 MB (it keeps 60 binary in fixed binary format, so the size should not grow beyond that):
```py
du -sch /var/log/sa
33M     /var/log/sa
33M     total
```
For CAP_XENSERVER_LOGS: Yes, I think not setting a max_size for CAP_XENSERVER_LOGS would simplify the code as it currently recalculates and updates its size, which may not be needed then. But we can keep that as it is now. If I understand this correctly, this can be done later when we clean up the code.